test/hooks_functional.js

```diff
@@ -79,10 +82,11 @@ function waitServerReady(port) {
     });
 }
 
-const layersAreInitialized = async () => {
-    await page.waitForFunction(() => view.mainLoop.scheduler.commandsWaitingExecutionCount() === 0
-        && view.mainLoop.renderingState === 0
-        && view.getLayers().every(layer => layer.ready), { timeout: 60000 });
+const layersInitialization = async (forceTimeOut = false) => {
+    await page.waitForFunction(forceTimeOut => (view.mainLoop.scheduler.commandsWaitingExecutionCount() === 0
+            && view.mainLoop.renderingState === 0
+            && view.getLayers().every(layer => layer.ready)
+            && !forceTimeOut), { timeout }, forceTimeOut);
```

Comment:
It seems that `forceTimeout` is never called with a `true` value, what is the purpose of this parameter? Moreover, you could use the [`signal`](https://pptr.dev/api/puppeteer.framewaitforfunctionoptions) parameter of `waitForFunction` (of type [`AbortSignal`](https://developer.mozilla.org/fr/docs/Web/API/AbortSignal)) to abort waiting for this function.